components/HMI/src/HumanInterface.cpp

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file HumanInterface.cpp
+ * @author Locha Mesh project developers (locha.io)
+ * @brief 
+ * @version 0.1.1
+ * @date 2019-12-18
+ * 
+ * @copyright Copyright (c) 2019 Locha Mesh project developers
+ * @license Apache 2.0, see LICENSE file for details
+ */
+
+#include "HumanInterface.h"
+#include <cstdint>
+#include <cstring>
+#include <esp_log.h>
+
+namespace hmi {
+static xQueueHandle gpio_evt_queue = nullptr;
+// ----- Initialization and Default Values -----
+/**
+ * @brief Construct a new OneButton object but not (yet) initialize the IO pin.
+ */
+Button::Button()
+{
+    _gpio_btn = static_cast<gpio_num_t>(-1);
+}
```

Comment:
Why this constructor exists? It creates an invalid _gpio_btn. Does it has an use?

Reply:
you right , I need to remove it